src/back/TicketManager/src/Core/TicketManager.Core.Services/Endpoints/Organizers/OrganizerDecideValidator.cs

```diff
@@ -0,0 +1,46 @@
+using FastEndpoints;
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using TicketManager.Core.Contracts.Organizers;
+using TicketManager.Core.Domain.Organizer;
+using TicketManager.Core.Services.Services.Mockables;
+using TicketManager.Core.Services.ValidationExtensions;
+
+namespace TicketManager.Core.Services.Endpoints.Organizers;
+
+public class OrganizerDecideValidator : Validator<OrganizerDecideRequest>
+{
+    private IServiceScopeFactory scopeFactory;
+    private MockableCoreDbResolver dbResolver;
+    
+    public OrganizerDecideValidator(IServiceScopeFactory scopeFactory, MockableCoreDbResolver dbResolver)
+    {
+        this.scopeFactory = scopeFactory;
+        this.dbResolver = dbResolver;
+
+        RuleFor(req => req.OrganizerId)
+            .MustAsync(IsIdPresent)
+            .WithCode(OrganizerDecideRequest.ErrorCodes.OrganizerNotInDatabase)
+            .MustAsync(IsUnverified)
+            .WithCode(OrganizerDecideRequest.ErrorCodes.OrganizerAlreadyVerified);
```

Comment:
You can specify custom messages here, because those generated for `Must` are not very precise.